libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,13 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+class Chef
+  class Resource
+    def sensitive(arg)
```

Comment:
Rubocop is suggesting 
```
sensitive(*)
```

for the definition. We are only defining the method if it doesn't already exist. 
How would adding nil be more passive?